docs/changelog.rst

```diff
@@ -32,7 +32,6 @@ We use the following annotations for classifying changes:
 0.4.0
 -----
 
-* |break| Change syntax to trigger a job replication, rename ``zrepl signal wakeup JOB`` to ``zrepl signal replication JOB``
```

Comment:
@problame not sure if dropping this line is the way to go. alternatively we could do a strikethrough and add something like 'postponed to a future release'
We definitely need to mention this revert in the RC2 release notes tho.